source/_integrations/co2signal.markdown

```diff
@@ -37,7 +37,12 @@ The Electricity Maps integration helps you understand the carbon intensity of yo
 ## Prerequisites
 
 To configure and use this integration, you need to obtain a free API key from Electricity Maps by signing up to the Free Tier product on the [Electricity Maps API Portal](https://electricitymaps.com/free-tier).
-Please be aware that the Free Tier API is limited to one location (called a zone). You need to specify the zone for your home location when creating your account.
+
+Please be aware that the Free Tier API is limited to one location, called a zone. A zone represents a specific electricity grid region. You need to select and specify the zone for your home location when creating your account. For example, the United States has multiple zones like `US-CENT-SWPP` (Southwest Power Pool), `US-CAR-DUK` (Duke Energy Carolinas), or `US-CAR-CPLE` (Duke Energy Progress East).
+
+To find your zone identifier, you can:
+- Check the [Electricity Maps app](https://app.electricitymaps.com) to see which zone covers your location.
+- Use the [Zones API reference](https://portal.electricitymaps.com/developer-hub/api/reference#zones) to look up zones by geographic coordinates.
```

Comment:
```suggestion
- Use the [Zones API reference](https://app.electricitymaps.com/developer-hub/api/reference#zones) to look up zones by geographic coordinates.
```

We are in process of migrating everything to app.electricitymaps.com

Currently the same site is hosted on both subdomains.